server/config.py

```diff
@@ -0,0 +1,42 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+from dotenv import load_dotenv
+load_dotenv()
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    CSRF_ENABLED = True
+    SECRET_KEY = 'this-really-needs-to-be-changed'
+    PG_COVID_USER = os.environ['PG_COVID_USER']
+    PG_COVID_PASSWORD = os.environ['PG_COVID_PASSWORD']
+    PG_COVID_HOST = os.environ['PG_COVID_HOST']
+    PG_COVID_PORT = os.environ['PG_COVID_PORT']
+    PG_COVID_DB_NAME = os.environ['PG_COVID_DB_NAME']
+    SQLALCHEMY_DATABASE_URI = "postgresql://%s:%s@%s:%s/%s" % (
+        PG_COVID_USER,
+        PG_COVID_PASSWORD,
+        PG_COVID_HOST,
+        PG_COVID_PORT,
+        PG_COVID_DB_NAME
+    )
+
+
+
+
+class ProductionConfig(Config):
+    DEBUG = False
+
+
+class StagingConfig(Config):
+    DEVELOPMENT = True
```

Comment:
Mind you just use the environment variables for that kind or configuration?
IMHO, an application must not be aware of its environment, just to its configuration